plugins/discourse-calendar/assets/javascripts/discourse/components/upcoming-events-list.gjs

```diff
@@ -95,8 +96,13 @@ export default class UpcomingEventsList extends Component {
     const data = {
       limit: this.count,
       before: moment().add(this.upcomingDays, "days").toISOString(),
+      after: moment().add(-2, "hours").toISOString(),
```

Comment:
I don't mind the add(-2) but ftr there's subtract: https://momentjs.com/docs/#/manipulating/subtract/